SU-HSDO/suhumsci
# Bugherd fixups

# READY FOR REVIEW

# Summary
- Increased gutter between paragraph cards
- Expanded area for hovering when adding a new paragraph below an existing one (plus icon)

# Needed By (Date)
- End of sprint (9/26)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that the spacing between cards (homepage is a good place to test) has increased
3. Ensure that when editing a page you can now hover above the plus icon and to the right of the paragraph tab without losing the ability to add a new paragraph item below

# Affected Projects or Products
- Archaeology
- suhumsci (theme and admin theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-357
- https://stanfordits.atlassian.net/browse/ARCH-383

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
